src/test_data/method_parameter_count_changed.output.ron

```diff
@@ -0,0 +1,77 @@
+[
+    {
+        "method_name": String("associated_function_with_a_parameter_added"),
+        "method_visibility": String("public"),
+        "name": String("StructWithMethods"),
+        "old_parameter_count": Uint64(0),
+        "path": List([
+            String("semver_tests"),
+            String("test_cases"),
+            String("parameter_count_changed"),
+            String("StructWithMethods")
+        ]),
+        "span_begin_line": Uint64(26),
+        "span_filename": String("src/test_cases/parameter_count_changed.rs"),
+        "visibility_limit": String("public")
```

Comment:
These outputs don't seem to have the `current_parameter_count` key-value which the `per_result_error_template` requires. I think you probably forgot an `@output` for it in the query.

Is there a way we could have caught this bug automatically, rather than catching it in code review? In a way, catching bugs in code review is like a trapeze artist falling into the safety net during a performance: it's great that the safety net was there, but the actual objective was to not slip and fall at all.

Reply:
Wouldn't it be easy to for the error message be tested as well? In the test, after receiving the query results. It should be easy to check if the error message template can be instantiated, and there could also be some precise expected output as it is currently for the queries in the src/test_data

Reply:
That could certainly work — ideally combined with an assertion that the expected output for each query file is non-empty i.e. that an example row exists with which to test instantiating the error message. Testing just the instantiation is probably a good start — I don't think we have any particularly complex templating going on right now for which we might need more precise output testing.